services/webhook/tests/test_code_reviewer_dispatch.py

```
@@ -312,7 +312,8 @@ def test_inline_comment_body_includes_suggestion_block(monkeypatch):
         message="m", suggestion="add a None guard",
     )
     body = cr_dispatch._inline_comment_body(f)
-    assert "Suggested fix" in body and "add a None guard" in body
+    assert "**Fix**" in body and "add a None guard" in body
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)